### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by 2HEADSs**
update documentation and add new controller : "editDeleted"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,26 +151,27 @@ Update entry with ID `654651caf696083cab72ab1c` in the `events` collection:
 -   **Endpont:** `/user/editUserInfo/:id`
     Headers: `Content-Type: application/json`
     Body: JSON-formatted data
-    Changes: Only optional data of User(firstName, lastName, region, organizationName, phone). Admin can change also "isDeleted" and "role" property.
+    Changes: Only optional data of User(firstName, lastName, region, organizationName, phone).Admin and User itself can change the data, but only if the requester of changes is not deleted or is approved.
```
Comment (Getsov) [inline README.md:154]: Why deleted space before the start of the sentence "Admin..." ?

File: README.md [hunk 2]
```diff
@@ -152,26 +151,27 @@ Update entry with ID `654651caf696083cab72ab1c` in the `events` collection:
...
-    Changes: Admin and User itself can change password.
+    Changes: Admin and User itself can change password, but only if the requester of changes is not deleted or is approved.
     To fulfill the request, the user must send the old password, a new password and a repetition of the new password. The administrator must send a new password and repetition the new password.
 
 -   **Method:** `PUT`
 -   **Endpont:** `/user/editUserRole/:id`
     Headers: `Content-Type: application/json`
     Body: JSON-formatted data
-    Changes: Admin only can change the role of single user.
+    Changes: Admin only, if not deleted or is approved, can change the role of single user, but only if he is not deleted or is approved.
```
Comment (Getsov) [inline README.md:174]: Repeating

File: README.md (hunk 1 above)
Comment (2HEADSs) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (2HEADSs) [reply README.md]: I didn't express myself correctly.
Fixed

**Review by Getsov — APPROVED** [review]

**Pull request merged**